heartbeat/redis

```diff
@@ -324,6 +324,7 @@ function start() {
 
 	[[ ! -d "$REDIS_RUNDIR" ]] && mkdir -p "$REDIS_RUNDIR"
 	chown -R "$REDIS_USER" "$REDIS_RUNDIR"
+	restorecon -Rv "$REDIS_RUNDIR"
```

Comment:
You're right, I'll change the pull request. What's the preferred method here ? amend and force push on the same branch, push to a different branch and change PR base, other ?